test/index.test.js

```diff
@@ -82,6 +82,7 @@ describe('Index Tests', () => {
               message: 'coralogix: flushing 1 pending requests...',
               level: 'info',
               timestamp: '2022-10-25T14:26:45.982Z',
+              logStream: '2022/10/25/[663]877ef64aed7c456086d40a1de61a48cc',
```

Comment:
messages in Coralogix would include this information which can easily be queried for in AWS CloudWatch